pkg/apis/core/types.go

```diff
@@ -3813,6 +3829,32 @@ type PodSecurityContext struct {
 	// Note that this field cannot be set when spec.os.name is windows.
 	// +optional
 	AppArmorProfile *AppArmorProfile
+	// seLinuxChangePolicy defines how the container's SELinux label is applied to all volumes used by the Pod.
+	// It has no effect on nodes that do not support SELinux or to volumes does not support SELinux.
+	// Valid values are "MountOption" and "Recursive".
+	//
+	// "Recursive" means relabeling of all files on all Pod volumes by the container runtime.
+	// This may be slow for large volumes, but allows mixing privileged and unprivileged Pods sharing the same volume on the same node.
+	//
+	// "MountOption" mounts all eligible Pod volumes with `-o context` mount option.
+	// This requires all Pods that share the same volume to use the same SELinux label.
+	// It is not possible to share the same volume among privileged and unprivileged Pods.
+	// Eligible volumes are in-tree FibreChannel and iSCSI volumes, and all CSI volumes
+	// whose CSI driver announces SELinux support by setting spec.seLinuxMount: true in their
+	// CSIDriver instance. Other volumes are always re-labelled recursively.
+	// "MountOption" value is allowed only when SELinuxMount feature gate is enabled.
+	//
+	// If not specified and SELinuxMount feature gate is enabled, "MountOption" is used.
```

Comment:
I understand that it will make catching impacted workloads easier, but changing the default behavior based on which feature gates are enabled seems like a confusing user experience. Would it be safer to just have workloads opt-in on their own?

Reply:
We discussed that with @deads2k. Our numbers show 1.15% of OpenShift clusters have at least one pod (Deployment, StatefulSet) that may require opt-out. 98.85%  of cluster won't need to do anything and can just upgrade.

(Rounding up), we considered that it makes more sense to let the 2% to force them to opt-out (and only few pods in these clusters need to opt-out) than 98% to opt-in all their pods.

Reply:
We discussed this with @deads2k. Current OpenShift numbers show that 1.15% of clusters have at least one Pod (Deployment, StatefulSet) that would need an opt-out. 98.85% of clusters can just upgrade and get faster relabeling. 

Exaggerating the number, we think it's better to warn 2% of clusters to opt-out before upgrade, than let 98% of clusters to opt-in explicitly.

Reply:
Ok, can you update the release notes with some more warnings and/or a link to a page where there's more information on how they can test/check?

Reply:
Edited